css/site.styl

```diff
@@ -27,21 +27,27 @@ normalize()
 
 // VARIABLES
 
-$color-black-soft = rgb(14, 14, 14)
-$color-white-soft = rgb(253, 253, 253)
+$color-black = rgb(0, 0, 0)
+$color-white = rgb(255, 255, 255)
 
 // ******************************************************
 
 // BASIC
 
 html
-  background-color: $color-white-soft
-  color: $color-black-soft
+  background-color: $color-white
+  color: $color-black
 
 img
   max-width: 100%
   height: auto
 
+.mobile-only
+  display: initial !important 
```

Comment:
well i dunno sometimes it 'could' but perhaps we should be making sure not to. I actually don't know why important is considered so evil?